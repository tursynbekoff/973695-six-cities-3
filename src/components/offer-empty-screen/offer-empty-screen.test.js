import React from "react";
import renderer from "react-test-renderer";
import EmptyScreen from "./offer-empty-screen.jsx";

const currentCity = `Amsterdam`;

it(` EmptyScreen render has problems`, () => {

  const tree = renderer
      .create(
          <EmptyScreen
            currentCity={currentCity}
          />
      ).toJSON();

  expect(tree).toMatchSnapshot();
});

