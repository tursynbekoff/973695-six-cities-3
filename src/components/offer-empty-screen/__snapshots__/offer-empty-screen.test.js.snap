// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` EmptyScreen render has problems 1`] = `
<div
  className="cities__places-container cities__places-container--empty container"
>
  <section
    className="cities__no-places"
  >
    <div
      className="cities__status-wrapper tabs__content"
    >
      <b
        className="cities__status"
      >
        No places to stay available
      </b>
      <p
        className="cities__status-description"
      >
        We could not find any property availbale at the moment in 
        Amsterdam
      </p>
    </div>
  </section>
  <div
    className="cities__right-section"
    style={
      Object {
        "backgroundImage": "url(../img/no-places@2x.png)",
        "backgroundPosition": "right center",
        "backgroundSize": "100% auto",
      }
    }
  />
</div>
`;
